### PR TITLE
Update Hacking.md

### PR DESCRIPTION
**Pull request by Ambg05**

### CODE REVIEW

**Comment by DinisCruz** [comment]
not sure I like the 'As I mentioned in my introduction' part (it can get quite repetitive)

**Comment by Ambg05** [comment]
I think it's better to acknowledge the repetition, as in this chapter you elaborate on it, but I can delete 'As I mentioned', just say the word!

File: manuscript/Hacking.md [hunk 1]
```diff
@@ -10,12 +10,14 @@ type: content
 
 It is important to state that hackers are the good guys.
 
-To 'Hack' is to solve problems, to find innovative solutions in a creative way.
+As I mentioned in my introduction, to 'hack' means to solve problems, to find innovative solutions in a creative way.
```
Comment (DinisCruz) [inline manuscript/Hacking.md]: This doesn't fell right to me, delete 'As I mentioned in my introduction, '

File: manuscript/Hacking.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ type: content
...
 It is important to state that hackers are the good guys.
 
-To 'Hack' is to solve problems, to find innovative solutions in a creative way.
+As I mentioned in my introduction, to 'hack' means to solve problems, to find innovative solutions in a creative way. 
 
 The press abuses the term 'hacker'. Instead, they should qualify the word by saying 'Malicious Hackers' or 'Cyber Attackers' or 'Cyber Criminals'.
 
 The internet and most of the technology we use today was dreamed of, and created by, hackers.
 
+There is a very [interesting news clip](http://www.bbc.co.uk/news/video_and_audio/features/magazine-38175202/38175202) that reports on the story of Abel Cruz, a Peruvian entrepreneur who worked out how to provide water, by means of fog nets, to poor people living off the grid in the hills outside Lima. Abel contacted fog net experts on the internet to make his idea a reality. Sixty fog nets now provide water to 250 families at a very low cost. This creative and innovative solution has transformed their lives.
```
Comment (DinisCruz) [inline manuscript/Hacking.md:21]: add title to this paragraph. Something like 'Real world hacks'

File: manuscript/Hacking.md [hunk 3]
```diff
@@ -45,7 +47,12 @@ This generation of hackers will create technology to sell PT products around the
  
 The generation that was hacking Linux boxes while we were in university is the same generation that is solving fundamental problems today. 
 
-We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one. There is a factory in Portugal where this can be done.
+We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one (see https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit/). There should be a factory in Portugal that can do this.  
+
+In fact, there should be software and technology companies in every city and town in Portugal. Software development and application security lend themselves to remote work, with regular commutes to a central office for meetings etc.
+
+(See for example how many Portuguese security researchers participate in global/world wide bug bounties.)
```
Comment (DinisCruz) [inline manuscript/Hacking.md]: Remove brackets
the Evidence for from is at https://pages.bugcrowd.com/hubfs/PDFs/state-of-bug-bounty-2016.pdf

image

Should we include this image in book?

File: manuscript/Hacking.md [hunk 4]
```diff
@@ -45,7 +47,12 @@ This generation of hackers will create technology to sell PT products around the
  
 The generation that was hacking Linux boxes while we were in university is the same generation that is solving fundamental problems today. 
 
-We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one. There is a factory in Portugal where this can be done.
+We need to make sure that this new generation knows what a Raspberry Pi is, knows what is inside, and knows how to build one (see https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit/). There should be a factory in Portugal that can do this.
```
Comment (DinisCruz) [inline manuscript/Hacking.md]: Links don't print out well. 
Replace with a comment about how it has been a successful commercial success with  [Ten Millionth Raspberry Pi](https://www.raspberrypi.org/blog/ten-millionth-raspberry-pi-new-kit)  sold, which ironically is the population of Portugal. The Raspberry Pi team has made an effort to build them in UK (which is where the reference to a Factory in PT make sense)

File: manuscript/Hacking.md (hunk 1 above)
Comment (Ambg05) [reply manuscript/Hacking.md]: Done.

File: manuscript/Hacking.md (hunk 2 above)
Comment (Ambg05) [reply manuscript/Hacking.md:21]: Done.

File: manuscript/Hacking.md (hunk 4 above)
Comment (Ambg05) [reply manuscript/Hacking.md]: Done.

File: manuscript/Hacking.md (hunk 3 above)
Comment (Ambg05) [reply manuscript/Hacking.md]: I just tried to but I have read-only Adobe at the moment, so I can't copy the image to the book. If you want to add it in I will reference it.  I think it's a good idea to include it.

**Comment by DinisCruz** [comment]
Great set of changes, thanks

**Pull request merged**